test/e2e/performanceprofile/functests/6_mustgather_testing/test_suite_mustgather_test.go

```diff
@@ -1,46 +1,53 @@
 package pao_mustgather
 
 import (
-	"context"
+	"fmt"
 	"os"
-	"path/filepath"
+	"os/exec"
 	"testing"
-	"time"
 
+	"github.com/onsi/ginkgo"
 	. "github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
 	. "github.com/onsi/gomega"
 
-	"k8s.io/apimachinery/pkg/api/errors"
-
 	ginkgo_reporters "kubevirt.io/qe-tools/pkg/ginkgo-reporters"
 
-	testutils "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils"
-	testclient "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/client"
 	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/junit"
-	testlog "github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/log"
-	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/namespaces"
 )
 
 var _ = BeforeSuite(func() {
-	Expect(testclient.ClientsEnabled).To(BeTrue(), "package client not enabled")
-	// create test namespace
-	err := testclient.Client.Create(context.TODO(), namespaces.TestingNamespace)
-	if errors.IsAlreadyExists(err) {
-		testlog.Warning("test namespace already exists, that is unexpected")
-		return
+	By("Looking for oc tool")
+	ocExec, err := exec.LookPath("oc")
+	if err != nil {
+		fmt.Fprintf(ginkgo.GinkgoWriter, "Unable to find oc executable: %v\n", err)
+		Skip(fmt.Sprintf("unable to find 'oc' executable %v\n", err))
 	}
-	Expect(err).ToNot(HaveOccurred())
+
+	mgImage := "quay.io/openshift-kni/performance-addon-operator-must-gather"
+	mgTag := "4.11-snapshot"
```

Comment:
version should be a top-level constant to make it trivial to update for newer branches